apps/backend/src/datasources/stfc/StfcProposalDataSource.ts

```diff
@@ -61,6 +61,7 @@ export default class StfcProposalDataSource extends PostgresProposalDataSource {
       .from('proposal_table_view')
       .whereIn('proposal_pk', proposals)
       .orderBy('proposal_pk', 'desc')
+      .where('proposal_status_id', '<>', '9')
```

Comment:
Can this be made to be a filter rather than hardcoding it we can pass a list of status which must be excluded